src/core_ocean/analysis_members/mpas_ocn_eddy_product_variables.F

```diff
@@ -224,7 +229,9 @@ subroutine ocn_compute_eddy_product_variables(domain, timeLevel, err)!{{{
                velocityMeridionalSquared(k,iCell) = velocityMeridional(k,iCell)**2
                velocityZonalTimesTemperature(k,iCell) = velocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
                velocityMeridionalTimesTemperature(k,iCell) = velocityMeridional(k,iCell)*activeTracers(index_temperature,k,iCell)
-            end do
+               velocityZonalTimesTemperature_GM(k,iCell) = GMBolusVelocityZonal(k,iCell)*activeTracers(index_temperature,k,iCell)
```

Comment:
I don't see where GMBolusVelocityZonal is being calculated.  I think this should be in mpas_ocn_time_integration_split, did you perhaps forget to commit changes to that file?  I would expect this code to give all zeros for this and the meridional array.

Reply:
this is my mistake, ignore the comment.  The BolusZonal is already in ocn_diagnostics